components/editor/modules/meta/FBPreview.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import { css } from 'glamor'
+
+const styles = {
+  fbContainer: css({
+    backgroundColor: '#fff',
+    color: '#000',
+    width: 476
+  }),
+  fbImage: css({
+    width: 476,
+    height: 249,
+    backgroundSize: 'cover'
+  }),
+  fbText: css({
+    padding: '10px 12px',
+    maxHeight: 120,
+    overflow: 'hidden'
+  }),
+  fbTitle: css({
+    fontFamily: 'Georgia, serif',
+    fontSize: 18,
+    fontWeight: 500,
+    lineHeight: '22px',
+    maxHeight: 110,
+    overflow: 'hidden',
+    marginBottom: 5,
+    wordWrap: 'break-word'
+  }),
+  fbDescription: css({
+    fontFamily: 'sans-serif',
+    fontSize: 12,
+    lineHeight: '16px',
+    maxHeight: 80,
+    overflow: 'hidden'
+  }),
+  fbDomain: css({
+    fontFamily: 'sans-serif',
+    fontSize: 11,
```

Comment:
Just a question:
Why "fontSize: 11" but "lineHeight: '11px'" (including "px")? Could you also just do "lineHeight: 11", or will glamor complain?

Reply:
`line-height` accepts unitless number – interpreted as multipler of `font-size`. So glamor would translate 11 to unitless 11 (11 * 11 = 121px)—when css properties don't accept unitless glamor automatically adds `px`—`fontSize: 11` becomes `font-size: 11px`